components/Cart/models.rst

```diff
@@ -14,13 +16,15 @@ The cart is represented by **Cart** instance. It inherits all the properties fro
 
 .. note::
 
-    This model implements ``CartInterface``
+    This model implements :ref:`component_cart_model_cart-interface`
+
+.. _component_cart_model_cart-item:
 
 CartItem
 --------
 
-The items of the cart are represented by **CartItem** instance and it has not property but it inherits all of them from ``Sylius\Component\Order\Model\OrderItem``
+The items of the cart are represented by **CartItem** instance and it has no property but it inherits all of them from :ref:`component_order_model_order-item`
```

Comment:
``The items in the cart are represented by the **CartItem** instances. **CartItem** has no properties but it inherits from :ref:`component_order_model_order-item`. ``